configs/default/TEBS-PODRACERAIO.config

```diff
@@ -0,0 +1,84 @@
+# Betaflight / STM32F411 (S411) 4.2.6 Jan  5 2021 / 19:07:43 (a4b6db1e7) MSP API: 1.43
```

Comment:
Can you please split these into different pull requests, one per manufacturer? Thanks.

Reply:
Ok Mike, i will do 3 different PR, i already opened the Tmotor